install_config/aggregate_logging.adoc

```diff
@@ -468,81 +468,84 @@ endif::openshift-enterprise[]
 [[scaling-elasticsearch]]
 *Changing the Scale of Elasticsearch*
 
-If you need to scale up the number of Elasticsearch instances your cluster uses
-it is not as simple as changing the number of replicas. This is due to the nature
-of Persistent volumes and how Elasticsearch is configured to store its data and
-recover the cluster.
+If you need to scale up the number of Elasticsearch instances your cluster uses,
+it is not as simple as changing the number of replicas. This is due to the
+nature of persistent volumes and how Elasticsearch is configured to store its
+data and recover the cluster.
 
-Instead, you will need to create a Deployment Configuration for each replica.
+Instead, you will need to create a deployment configuration for each replica.
 
-The Deployer created templates during installation with the Elasticsearch
-configurations provided to it: `logging-es-template` and `logging-es-ops-template`
-if the Deployer was run with `ENABLE_OPS_CLUSTER=true`.
+The deployer created templates during installation with the Elasticsearch
+configurations provided to it: *logging-es-template* and
+*logging-es-ops-template* if the deployer was run with
+`*ENABLE_OPS_CLUSTER=true*`.
 
-The node quorum and recovery settings were initially set based on the `CLUSTER_SIZE`
-value provided to the deployer.  Since we are changing the cluster size, those
-values will need to be updated.
+The node quorum and recovery settings were initially set based on the
+`*CLUSTER_SIZE*` value provided to the deployer. Since the cluster size is
+changing, those values need to be updated.
 
-Prior to changing the number of replicas, to preserve log data the EFK stack should
-first be scaled down as described in steps 1-4
-link:../upgrading/manual_upgrades.html#manual-upgrading-efk-logging-stack[here].
+Prior to changing the number of replicas, the EFK stack should first be scaled
+down to preserve log data as described in
+link:../install_config/upgrading/manual_upgrades.html#manual-upgrading-efk-logging-stack[Upgrading
+the EFK Logging Stack].
 
-For the changes in the cluster template and the cluster Deployment Configs we will
-use the following values:
+For the changes in the cluster template and the cluster deployment
+configurations, use the following values:
 
-====
-`NODE_QUORUM` is the intended cluster size / 2 (rounded down) + 1.
-So for an intended cluster size of 5 the quorum would be 3.
+- `*NODE_QUORUM*` is the intended cluster size / 2 (rounded down) + 1. For an
+intended cluster size of 5, the quorum would be 3.
 
-`RECOVER_EXPECTED_NODES` is the same as the intended cluster size.
+- `*RECOVER_EXPECTED_NODES*` is the same as the intended cluster size.
 
-`RECOVER_AFTER_NODES` is the intended cluster size - 1.
-====
+- `*RECOVER_AFTER_NODES*` is the intended cluster size - 1.
+
+Edit the template of the cluster you are scaling up and update the three
+parameters as specified above.
```

Comment:
This might be out of order. The above seems step 2 of a procedure, but mentions to change the "values above". I'd suggest something like:

2. Edit the cluster template you are scaling up and change the parameters to the desired value:

(then list the values)